### ISSUE TITLE
Hai Leaks Response 6 blocked due to clearance issues

### ISSUE THREAD

**Issue by roadrunner56** [endless-sky/endless-sky#8325]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I'm beginning to play through Hai Reveal, and finished Hai Leaks Response 5. In that dialog it said that I could leave for a while and come back when I was ready to continue. However, once I get back to Hai space I cannot land on Hai-Home as it is clearance-locked. I cannot bribe my way onto the planet, effectively locking me out of HR. 

### Steps to Reproduce

1. Finish Hai Leaks Response 5
2. Leave Hai-Home and do other things for a while
3. Come back to Hai-Home
4. Fail to land as clearance blocks you from doing so.

### Expected Behavior

I expect to be able to land on Hai-Home now that I've become an established figure to the Hai government.

### Link to save file

[Todd Tucker.txt](https://github.com/endless-sky/endless-sky/files/10693059/Todd.Tucker.txt)

### Operating System

Linux: Kubuntu 22.04

### Game Source

ESLauncher2

### Game Version

Commit fad28d7

**Comment by RisingLeaf** [comment]
this should be easy to fix, shouldn't it?
just set reputation in a mission

**Comment by bene-dictator** [comment]
This should be fixed in #8243.

**Comment by roadrunner56** [comment]
Should this be closed now that #8243 has been merged?

**Comment by warp-core** [comment]
I don't know if that fixes it. It looks like the "5B" mission added there would probably fix it, so yes, I suppose.
If someone reports this issue again, we can reopen this or they can make a new issue.
